README.md

```diff
@@ -0,0 +1,230 @@
+# coingate-sdk
+
+[![npm package][npm-img]][npm-url]
+
+## Install
+
+```bash
+npm install coingate-sdk
+```
+
+## Getting Started
+
+You can sign up for a CoinGate account at https://coingate.com for production and https://sandbox.coingate.com for testing (sandbox).
+
+Please note, that for Sandbox you must generate separate API credentials on https://sandbox.coingate.com. API credentials generated on https://coingate.com will not work for Sandbox mode.
+
+Usage of CoinGate package looks like:
+
+```ts
+const client = new Client('YOUR_API_TOKEN');
+```
+
+In order, to use sandbox mode, you need to set second parameter to `true`.
+
+```ts
+const client = new Client('YOUR_API_TOKEN', true);
+```
+
+If you plan to use Public API endpoints only, authentication is not required.
+
+```ts
+const client = new Client();
+
+// if needed you can set configuration parameters later
+client.setApiKey('YOUR_API_TOKEN');
+client.setEnvironment('sandbox');
+```
+
+### Order
+
+#### Create order
+
+Create order at CoinGate and redirect shopper to invoice (payment_url).
+
+```ts
+const data = {
+  order_id: 'YOUR-CUSTOM-ORDER-ID-123', // optional
+  price_amount: 89.99,
+  price_currency: 'GBP',
+  receive_currency: 'USD',
+  title: 'My first order', // optional
+  description: 'Amazon gift card', // optional
+  callback_url: 'https://example.com/payments?token=thiSisExAmPlE1o2k3a4y5', // optional
+  cancel_url: 'https://example.com/cart', // optional
+  success_url: 'https://example.com/account/orders', // optional
+  token: 'Your custom token', // optional
+  purchaser_email: 'example.guy@xmpl.com' // optional
+};
+
```

Comment:
document that async should be used

```
(async () => {
try {
  const order = await client.order.createOrder(data);
} catch (error) {
  // Oops... Something went wrong...
  console.error(error);
}
})();
```
or that can be used with promises
`client.order.createOrder(data).then(order => console.log(order)).catch(error => console.log(error))`